app/view/state/__init__.py

```diff
@@ -42,11 +44,13 @@ def getStates():
     state=State()
 
     result=state.get_States()
-
+    
     if result:
-        return jsonify({"message": "states displayed successfully", "status": 201,"result":result})
+        response=make_response({"message": "state displayed successfully", "status": 201,"result":result},201)
+        return response
     else:
-        return jsonify({"message": "states not displayed", "status": 400})
+        response=make_response({"message": "states not displayed", "status": 400},400)
```

Comment:
- Why `400`?

Reply:
ill implement error handling so it makes more sense

Reply:
What http status code is appropriate when no data is returned from the server? Like when the table is empty?